src/github.com/getlantern/fronted/cache.go

```diff
@@ -7,13 +7,12 @@ import (
 	"time"
 )
 
-const (
+var (
 	maxAllowedCachedAge = 24 * time.Hour
 	maxCacheSize        = 1000
 	cacheSaveInterval   = 5 * time.Second
-)
 
-var (
+	// Nil value indicates end of cache filling
```

Comment:
I know the linter complains, but I like the clarity of emphasizing that we're using a nil value as our sentinel.